MusicTaste/MusicTaste/View Controllers/EditViewController.h

```diff
@@ -0,0 +1,15 @@
+#import <UIKit/UIKit.h>
+#import "Parse/Parse.h"
+
+NS_ASSUME_NONNULL_BEGIN
+@protocol EditViewControllerDelegate
+
+@end
```

Comment:
Since the protocol doesn't specify any requirements, is it needed?

Reply:
No it wouldn't deleted the unnecessary protocol. Thank you for the heads up!